<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>filter</class>
 <widget class="QWidget" name="filter">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1058</width>
    <height>626</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="2" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="2">
      <widget class="QScrollArea" name="filterArea">
       <property name="widgetResizable">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>437</width>
          <height>308</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QScrollArea" name="edgeArea">
       <property name="widgetResizable">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>438</width>
          <height>308</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QScrollArea" name="oriArea">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="widgetResizable">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>438</width>
          <height>308</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QScrollArea" name="grayArea">
       <property name="widgetResizable">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>437</width>
          <height>308</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>1260</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="select_img">
       <property name="minimumSize">
        <size>
         <width>74</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="edge">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>200</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>200</height>
        </size>
       </property>
       <property name="title">
        <string>Edge Detection</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>251</width>
          <height>171</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="leftMargin">
            <number>20</number>
           </property>
           <property name="rightMargin">
            <number>20</number>
           </property>
           <item>
            <widget class="QRadioButton" name="robortsButton">
             <property name="text">
              <string>Roborts_1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="robortsButton_2">
             <property name="text">
              <string>Roborts_2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="prewittButton">
             <property name="text">
              <string>Prewitt_x</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="prewittButton_2">
             <property name="text">
              <string>Prewitt_y</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="sobelButton">
             <property name="text">
              <string>Sobel_x</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="sobelButton2">
             <property name="text">
              <string>Sobel_y</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="blur">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>120</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>120</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Noise Reduction</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>251</width>
          <height>96</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QRadioButton" name="gaussianButton">
             <property name="text">
              <string>Guassian</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="medianButton">
             <property name="text">
              <string>Median</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Kernel</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Sigma</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="spinBox">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>9</number>
               </property>
               <property name="singleStep">
                <number>2</number>
               </property>
               <property name="value">
                <number>5</number>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBox">
               <property name="minimum">
                <double>0.100000000000000</double>
               </property>
               <property name="maximum">
                <double>3.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.200000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="tableView">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>250</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>250</height>
        </size>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="autoScroll">
        <bool>false</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="textElideMode">
        <enum>Qt::ElideMiddle</enum>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>50</number>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>30</number>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>50</number>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>30</number>
       </attribute>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>1260</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
